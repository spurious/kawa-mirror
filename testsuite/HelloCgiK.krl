[(response-content-type "text/xml")
]<p>The request URL was: [(request-url)]</p>
<p>[(let ((query (request-query-string)))
    (if query
      ]There was no query string.[
      (begin ]The query string was: [query)))]</p>
