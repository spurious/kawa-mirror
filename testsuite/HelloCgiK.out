Content-Type: text/xml

<p>The request URL was: http://localhost/HelloCgiK</p>
<p>The query string was:  false</p>
