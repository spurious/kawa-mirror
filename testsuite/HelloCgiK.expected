Content-Type: text/xml

<p>The request URL was: http://localhost/HelloCgiK</p>
<p>There was no query string.</p>
